src/lib/api/loans/serializer.js

```diff
@@ -27,6 +27,25 @@ function serializeResponse(hit) {
   return result;
 }
 
+function serializeEmailResponse(hits) {
+  let results = [];
+  hits.hits.forEach((email) => {
```

Comment:
using `.map()` would simplify this implementation. 
Are you sure there isn't a simpler way of building a json object than assigning each key one by one ?